src/daos/sql-query-builder.php

```diff
@@ -112,15 +126,17 @@ public function or_where( $string ) {
 
   /**
    * Adds a variable to be bound to in the prepared statement
+   *
    * @param $name : string
    * @param $value : object
    */
-  public function bind_value( $name, $value ) {
-    $this->values_to_bind[ $name ] = $value;
+  public function bind_value( $name, $value, $data_type = \PDO::PARAM_STR ) {
```

Comment:
agreed, we don't want to specify a default parameter since php will use the data type. So if we pass in an integer and don't specify an data type then we should let the PDO library figure it out.